config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
+#For detail on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  scope "(:locale)", locale: /en/ do
+    root "static_pages#home"
+
+    get "static_pages/home"
+    get "static_pages/help"
+  end
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```

Comment:
remove all comments in this file
```suggestion
  scope "(:locale)", locale: /en/ do
    root "static_pages#home"

    get "static_pages/home"
    get "static_pages/help"
  end
```

Reply:
done